README.md

```diff
@@ -2,6 +2,10 @@
 
 Check your project for common sources of contributor friction.
 
+#### For the Non-Developer
+
+Flint checks if your project's folder contains the proper files and structure to allow potential contributors to understand: 1) what the project's mission is, 2) how to contribute, 3) under what conditions the project can be used, and 4) how to install the project.
```

Comment:
I like this. Two small suggestions. Could you swap:

> what the project's mission is

with

> the project's goals

... and 

> under what conditions the project can be used

with 

> usage guidelines

I think those changes make a clearer connection back to "understand" which introduces the series.